integration-tests/scripts/create-github-repo.sh

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env bash
+#
+# Summary:
+#   Creates a new GitHub repository using the GitHub REST API.
+#   Can create repositories in user account or organizations.
+#
+# Parameters:
+#   $1: repo_name         - The name of the repository to create (just the name, not owner/name)
+#                          OR full name like "org/repo" to create in an organization.
+#   $2: description       - Optional description for the repository.
```

Comment:
They are test repos that get removed when the test finishes. Why do we have to set descriptions for them?

Reply:
removed setting a description